### DISCUSSION TITLE
Brain Douglas - Open Sauced

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#238] [Announcements]
- https://opensauced.pizza
- https://twitter.com/bdougieYO
- https://twitter.com/saucedopen
- https://app.opensauced.pizza/user/bdougie
- https://github.com/bdougie
- https://www.linkedin.com/in/brianldouglas/

## Questions

- [x] You worked at GitHub for a long time. What did you work on and what was it like?
- [x] You told me that you had the idea for opensauced while at GitHub. Did you try to get it going as a feature there? What made you decide to pursue this full time?

#### Devs

- [x] What is open sauced? How does the platform aim to help devs?
- [x] How might open sauced help me get paid to be a freelance OSS dev? Are the any other ways it helps me get recognized?
- [x] Do you think a platform like this is needed for use to solve OSS funding?
- [x] The platform seems to help devs find projects to work on, is the opposite true? Can the platform help find maintainers for important code that's gone unmaintained?

#### Orgs

- [x] While the platform advertises itself as a tool for normal devs to find projects it seem like much more of an enterprise play. How does open sauced make money?
- [x] What types of data insights will a company be able get out of the platform?

#### Social Platform

- [x] The app seems almost like GitHub and twitter combined. Is there plans to also make this somewhat of a social network?

### Future

- [x] What's your spiciest dev take?
- [x] What are future plans/features for the platform?
- [x] What's the future of open source?

**Comment by bdougie** [comment]
Tooltips for me (so I remember)

- [Posthog](https://posthog.com/)
- [RustGPT](https://chat.openai.com/g/g-2LFEDLGgS-ferris-the-crab)

**Comment by hipstersmoothie** [comment]
- https://www.melt-ui.com/docs/introduction
- https://fleet.so/context

**Comment by just-be-dev** [comment]
- https://www.automa.site/
- https://svelteflow.dev/
